base/compiler/ssair/show.jl

```diff
@@ -31,11 +31,17 @@ function Base.show(io::IO, cfg::CFG)
     end
 end
 
-function print_stmt(io::IO, idx::Int, @nospecialize(stmt), used::BitSet, maxlength_idx::Int, color::Bool, show_type::Bool)
+function print_stmt(io::IO, idx::Int, @nospecialize(stmt), used::BitSet, maxlength_idx::Int, color::Bool, show_type::Bool,
+                    unstable_ssa::Union{Nothing, BitSet} = nothing)
     if idx in used
         idx_s = string(idx)
         pad = " "^(maxlength_idx - length(idx_s) + 1)
-        print(io, "%", idx_s, pad, "= ")
+        if unstable_ssa != nothing && idx in unstable_ssa
+            printstyled(io, "%", idx_s; color=:light_red, bold=true)
```

Comment:
Does this still print correctly for green (i.e. new) nodes?

Seems like it might not be respecting the `color::Bool` argument

Reply:
AFAIK, color was not used previously. I think we can remove color::Bool argument

Reply:
? It's used just below this on line 44/50

Reply:
Sorry, I did'nt explain it correctly. I mean that color is only check there, but if some color is defined, it is not use in any function as an argument, IINM

Reply:
I think it is solved, if I understood the issue.

Reply:
@topolarity